Source/WebCore/bindings/js/JSRemoteDOMWindowCustom.cpp

```diff
@@ -38,6 +38,10 @@ bool JSRemoteDOMWindow::getOwnPropertySlot(JSObject* object, JSGlobalObject* lex
     if (std::optional<unsigned> index = parseIndex(propertyName))
         return getOwnPropertySlotByIndex(object, lexicalGlobalObject, index.value(), slot);
 
+    // FIXME (rdar://115751655): This should be replaced with a same-origin check between the active and target document.
+    if (propertyName.isSymbol())
```

Comment:
Could we limit this to just $vm somehow so it's more clear why we added this hack?